spec/README-compat.md

```diff
@@ -0,0 +1,84 @@
+
+# Wasm Image Specification v0.0.0
+
+## Introduction:
+
+This document describes a varient of [Wasm Artifact Image Specification](README.md), which leverages the OCI "compatible" media type. Here, we omit definition and terminology explained in [Wasm Artifact Image Specification](README.md). 
+
+We call this variant "compat", and the spec in [Wasm Artifact Image Specification](README.md) "oci".
+
+## Description
+
+This *compat* variant makes use of `application/vnd.oci.image.layer.v1.tar+gzip` media type for layers, and is not based on custom OCI Artifcat media types. This way users can oeperate with standard tools such as docker, podman, buildah, etc.
```

Comment:
we should probably also allow the docker mediaType `application/vnd.docker.image.rootfs.diff.tar.gzip` - it's the same thing anyway, and it opens us up to supporting image building using `docker build`

Reply:
yeah good point. Let me check how we could embed annotation in the docker format first.

Reply:
OK we can use `"mediaType": "application/vnd.docker.distribution.manifest.v2+json"` in the docker manifest without using additional docker command. cool.

Reply:
implemented the here https://github.com/istio/istio/compare/master...mathetake:wasm-image-fetch?expand=1

Reply:
Fixed in https://github.com/solo-io/wasm/pull/265/commits/93e2c06c14be4fd4b91be98bbef39c3eac199437.